### ISSUE TITLE
Examples are not saved using Docker

### ISSUE THREAD

**Issue by FireChaos** [3b1b/manim#466]
I am unable to find the files while running the example
`python3 -m manim example_scenes.py SquareToCircle -l`
This is the output I'm getting, but there is no folder past /videos/
![screenshot_1](https://user-images.githubusercontent.com/7671689/53836942-f2595e00-3f99-11e9-9180-f6c0bb42fe22.png)
Using Docker, Windows 10.

Thanks in advance.

**Comment by eulertour** [comment]
I haven't been able to reproduce this. Can you give some more details? Are you bind-mounting manim in or cloning it from the container?

**Comment by FireChaos** [comment]
> 
> 
> I haven't been able to reproduce this. Can you give some more details? Are you bind-mounting manim in or cloning it from the container?

Cloning.
I should probably try to bind as well and report back

**Comment by eulertour** [comment]
No files are shared between the docker container and the host by default, so you'll have to bind mount a directory into the container to make the changes persist.

**Comment by FireChaos** [comment]
I've tried binding and replaced my absosulte math to my manim repo from this instruction:
`docker run -itv /absolute/path/to/your/local/manim/repo:/root/manim eulertour/manim`

Docker asked for my system's password, which I included and can now access my hard drive.
Running the example, I get:
`\path_to_python3\ : no module named 'manim'.

So I installed manim with 'pip3 install manim'
However, running the example now prints:
`No module named manim.__main__; 'manim' is a package and cannot be directly executed.

I've made sure that manim is installed, am I missing something?
Am I skipping something?
I'm new to this, so sorry for the inconvinience.
